### ISSUE TITLE
NEOGEO doesn't see 7z files

### ISSUE THREAD

**Issue by roo1234** [cizia64/CrossMix-OS#100]
I have 12 zip and the remaining 100s are 7z. Only the 12 zips are listed in the menu, despite the Emu config.json in NEOGEO having the 7z entry correct.

**Comment by cizia64** [comment]
refresh roms ?

**Comment by roo1234** [comment]
Yeah tried that, now confirming that refresh doesnt seem to be working, in menu button or settings, the same. File number in each emu folder do not reset.

**Comment by cizia64** [comment]
I can't reproduce on my side...
